ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -163,18 +163,15 @@ mkShelleyLedgerConfig ::
   -> EpochInfo (Except HardFork.PastHorizonException)
   -> MaxMajorProtVer
   -> ShelleyLedgerConfig era
-mkShelleyLedgerConfig genesis transCtxt epochInfo mmpv =
+mkShelleyLedgerConfig genesis transCtxt epochInfo _mmpv =
```

Comment:
Can we remove this parameter?

Reply:
I think we can. It'll inflate the diff a bit, but that's the only downside I foresee.

Reply:
Or we can leave it as per https://github.com/IntersectMBO/ouroboros-consensus/pull/1262/files#r1797178683

Reply:
This particular parameter is unrelated to diffusion pipelining, so it can definitely be removed.